protocols/imap/src/test/java/org/apache/james/imap/processor/GetAnnotationProcessorTest.java

```diff
@@ -0,0 +1,233 @@
+/****************************************************************
```

Comment:
We cannot verify the result from this test so I don't think that is need. It should be in the MPT I think.